### ISSUE TITLE
Internal server error: compiler version

### ISSUE THREAD

**Issue by tarrencev** [eqlabs/pathfinder#571]
We've been running into this error on certain function calls with Pathfinder v0.3.3

`Internal error: {'contract_definition': {'program': {'compiler_version': ['Unknown field.']}}}`

Calling `symbol()` on `0x3fd3642190a6fc7064b52ca2162f98ec1da754c88823a21b3a214880366b686` on goerli

**Comment by Mirko-von-Leipzig** [comment]
I imagine this is because pathfinder is still running cairo-vm 0.9.1 and not 0.10 yet.

**Comment by gregoryguillou** [comment]
This is how to reproduce

```
curl -s -X POST -H 'Content-Type: application/json' \
  -d '{"jsonrpc":"2.0","id":"0","method":"starknet_call","params":[{"contract_address": "0x3fd3642190a6fc7064b52ca2162f98ec1da754c88823a21b3a214880366b686", "entry_point_selector": "0x216b05c387bab9ac31918a3e61672f4618601f3c598a2f3f2710f37053e1ea4", "calldata": [] }, "latest"]}' https://localhost:9545 

{
  "jsonrpc": "2.0",
  "error": {
    "code": -32603,
    "message": "Internal error: {'contract_definition': {'program': {'compiler_version': ['Unknown field.']}}}"
  },
  "id": "0"
}
```

**Comment by Mirko-von-Leipzig** [comment]
`compiler_version` is a new property that was added with starknet 0.10. What is likely happening is that the older cairo-vm that pathfinder is running internally does not know how to handle the property.

This should get fixed once we upgrade the cairo-vm to 0.10 (this is in progress).
